### ISSUE TITLE
Top bar styling inconsistencies (and contrast issues)

### ISSUE THREAD

**Issue by aviannamiller** [omeka-s-themes/foundation#88]
All issues stated below pertain to mobile views for the horizontal dropdown layout only.

- For all stylesheets, the search toggle has this extra padding to it:
  <img width="501" height="69" alt="Detail of top bar, which includes menu toggle, site title, and search toggle. Edges of search toggle are protruding outside the bar." src="https://github.com/user-attachments/assets/0cd81088-e813-4e3c-85b5-14467bf644eb" />
- Between widths of 640 and 1024px, the background color of the bar changes for both Default and Revolution, leading to contrast failures for the site title.
  <img width="774" height="46" alt="Detail of top bar. Site title in bar appears in faint white text against light beige textured background." src="https://github.com/user-attachments/assets/605a0ba8-40e2-4b06-b839-abeeb1f20cbb" />
  <img width="774" height="46" alt="Detail of top bar. Site title in bar appears in faint white text against light grey background." src="https://github.com/user-attachments/assets/be5b6aa7-da70-4333-9f4b-99e11ca9d0a0" />

**Comment by kimisgold** [comment]
Fixes for this are now available on the `a11y/mobile-header-contrast` branch.

**Comment by aviannamiller** [comment]
Changes are excellent, thanks!
